examples/routing_service/routing_service_with_security/dds_security_01/cert/example2RSECdsa.cnf

```diff
@@ -0,0 +1,10 @@
+prompt=no
+distinguished_name	= req_distinguished_name
+
+[ req_distinguished_name ]
+countryName=US
+stateOrProvinceName=CA
+localityName=Santa Clara
+organizationName=Real Time Innovations
+emailAddress=D2_RSECdsa@rti.com
+commonName=rsexampleECdsa
```

Comment:
We may want to use camelCase here, or even use spaces?

Reply:
use camelCase convention.